addons/interaction/CfgVehicles.hpp

```diff
@@ -312,6 +312,14 @@ class CfgVehicles {
                 statement = "";
                 showDisabled = 1;
                 icon = "";  // @todo
+
+                class GVAR(weaponAttachments) {
+                    displayName = "$STR_A3_CfgEditorSubcategories_EdSubcat_SideSlot0";
+                    condition = QGVAR(enableWeaponAttachments);
+                    exceptions[] = {"isNotInside", "isNotSwimming", "isNotSitting"};
+                    insertChildren = QUOTE(call DFUNC(getWeaponAttachmentsActions));
+                    modifierFunction = QUOTE(_this select 3 set [ARR_2(2, getText (configFile >> 'CfgWeapons' >> currentWeapon (_this select 0) >> 'picture'))];);
```

Comment:
Could move to function to avoid code in config, but leaving it here is also fine.